src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,43 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
```

Comment:
Use LocalDate instead of Date. The Date class is outdated and has been replaced with the more modern and robust java.time (JSR-310) package in Java 8. Here is some background why you should use it primarily: https://www.baeldung.com/migrating-to-java-8-date-time-api.